CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ UNRELEASED
 * [ [#1131](https://github.com/digitalfabrik/integreat-cms/issues/1131) ] Flush Cache of related objects when changing a tree
 * [ [#1242](https://github.com/digitalfabrik/integreat-cms/issues/1242) ] Add setting to activate Matomo tracking
 * [ [#1197](https://github.com/digitalfabrik/integreat-cms/issues/1197) ] Fix calculation of translation status
+* [ [#1251](https://github.com/digitalfabrik/integreat-cms/issues/1251) ] Fix change detection in page form
```

Comment:
Could you add the changelog to the "unreleased" section please? (this probably requires a rebase on develop to resolve the conflict with the recently merged PR)